lib/navigation.js

```diff
@@ -0,0 +1,51 @@
+// Navgation metalsmith plugin
+// builds a navigation object based on config file and folder structure
+
+// files: array containing information about every page
+// metalsmith: object containing global information such as meta data
+// done: function which must be called when the plugin has finished working
+
+const fs = require('fs')
+const path = require('path')
+const paths = require('../config/paths.json')
+const navigation = require('../config/navigation.json')
+
+module.exports = function () {
+  return function (files, metalsmith, done) {
+    // iterate main navigation item defined in navigation.json
+    for (let item in navigation) {
+      // if is not homepage look for subitems
+      if (navigation[item].url !== '') {
```

Comment:
An error check and console logging here might be good in case `url` doesn't exist - I noticed that if any of the items in `navigation.json` are missing the `url` property, the build fails and the error message is not very clear.

Reply:
Updated the conditional statement to check if the url both exists and is not equal to the home path. The initial condition covered both, but this will probably make it more explicit.